godot-core/src/private.rs

```diff
@@ -331,9 +331,11 @@ where
     let info: Arc<Mutex<Option<GodotPanicInfo>>> = Arc::new(Mutex::new(None));
```

Comment:
I might need to get rid of this too, investigating

Reply:
`GodotPanicInfo` might also be cfg-gated. Maybe see if it brings other unused symbols with it

Reply:
can we use a thread-local here instead? idk if that improves performance but maybe

Reply:
We could, but since now it's debug only, performance isn't much of importance I guess. Making it thread-local, what would be implications? Would it still work same? I don't really wanna change functionality of this much.